frontend/src/components/GoalCards/GoalsCardsHeader.js

```diff
@@ -71,6 +74,15 @@ export default function GoalCardsHeader({
     }
   }, [canMergeGoals, recipientId, regionId]);
 
+  useEffect(() => {
+    async function checkFeatureFlags() {
+      const flags = await getFeatureFlags();
+      setHasManualMarkGoalsSimilar(flags.includes('manual_mark_goals_similar'));
+    }
+
+    checkFeatureFlags();
+  }, []);
```

Comment:
Is an empty [] on use effect allowed by our linter rules?

Reply:
useEffect with empty deps is standard, and means "run only once" - what's the concern?

Reply:
NVM seems like its a one and done thing.